generator/transit_generator.cpp

```diff
@@ -125,6 +132,11 @@ void BuildTransit(string const & mwmPath, string const & transitDir)
   SerializeObject<Stop>(root, "stops", serializer);
   header.m_gatesOffset = base::checked_cast<uint32_t>(w.Pos() - startOffset);
 
+  auto const fillPedestrianFeatureIds = [](function<void(vector<Gate> &)> & handler)
+  {
+    // @TODO(bykoianko) |m_pedestrianFeatureIds| is not filled from json but should be calculated based on |m_point|.
+  };
+  UNUSED_VALUE(fillPedestrianFeatureIds);
```

Comment:
Why don't you pass it to its SerializeObject? The implementation is empty anyway.

Reply:
Done